core/src/main/java/org/arquillian/smart/testing/Logger.java

```diff
@@ -1,19 +1,22 @@
 package org.arquillian.smart.testing;
 
+import java.util.logging.ConsoleHandler;
 import java.util.logging.Level;
 
 import static java.lang.String.format;
-import static org.arquillian.smart.testing.LoggerConfigurator.enableDebugLogLevel;
+import static org.arquillian.smart.testing.Configuration.SMART_TESTING_DEBUG;
 
 public class Logger {
 
     private final java.util.logging.Logger jul;
 
     private static final String PREFIX = "%s: Smart-Testing - ";
 
+    private boolean mavenDebugLogLevel = false;
+
     private Logger(java.util.logging.Logger logger) {
         this.jul = logger;
```

Comment:
If everything goes to `System.out` now why do we still need instance of the logger?